src/cli/utils/flags.go

```diff
@@ -65,6 +65,31 @@ func AddUserFlag(cmd *cobra.Command) {
 	cobra.CheckErr(cmd.MarkFlagRequired(UserFN))
 }
 
+// AddSiteIDFlag adds the --site-id flag, which accepts site ID values.
+// This flag is hidden, since we expect users to prefer the --site url
+// and do not want to encourage confusion.
+func AddSiteIDFlag(cmd *cobra.Command) {
+	fs := cmd.Flags()
+
+	// note string ARRAY var.  IDs naturally contain commas, so we cannot accept
+	// duplicate values within a flag declaration.  ie: --site-id a,b,c does not
+	// work.  Users must call --site-id a --site-id b --site-id c.
+	fs.StringArrayVar(
+		&Site,
+		SiteIDFN, nil,
+		//nolint:lll
+		"Backup data by site ID; accepts '"+Wildcard+"' to select all sites.  Args cannot be comma-delimited and must use multiple flags.")
```

Comment:
idk, word-smithing here but "multiple flags" makes it sound like there should be another (distinct) flag to use?
```suggestion
		"Backup data by site ID; accepts '"+Wildcard+"' to select all sites.  Args cannot be comma-delimited and must use multiple flag instances.")
```